practice-app/app/block.py

```diff
@@ -11,4 +11,4 @@
 @block_api.route('/api/v1.0/<int:user_id>/blocked-users', methods=['GET'])
 def get_blocked_users(user_id):
     blocked_users = session.query(Blocking).filter(Blocking.blockingID == user_id).all()
-    return jsonify({"blockedIDs": list(blocked_users)}), 200
+    return jsonify({col.name: str(getattr(blocked_users, col.name)) for col in blocked_users.__table__.columns}), 200
```

Comment:
Since this endpoint is for getting the users blocked by the user with given id, you don't need to return all Blocking object. Also, you tried to  access object attributes from list not the object's itself. So, here is my suggestion:
```suggestion
    return jsonify({"blockedIDs": [user.blockedID for user in blocked_users]}), 200
```

Reply:
Thank you for your review Kerem. I think you are correct, I am implementing your suggestion.